pom.xml

```diff
@@ -1232,6 +1232,11 @@
                 <version>${project.version}</version>
                 <type>test-jar</type>
             </dependency>
+            <dependency>
+                <groupId>${project.groupId}</groupId>
+                <artifactId>james-server-mailrepository-memory</artifactId>
```

Comment:
james-server-webadmin-mailrepository ?

Reply:
?

Reply:
No the memory implem is used in tests

Reply:
And the depandancy to james-server-webadmin-mailrepository is added later on (for guice bindings)

Reply:
Ok